doc/source/main-documentation/qibolab.rst

```diff
@@ -633,6 +633,14 @@ The most important instruments are the controller, the following is a table of t
 Zurich Instruments
 ^^^^^^^^^^^^^^^^^^
 
+``Qibolab`` has been tested with the following `instrument cluster <https://www.zhinst.com/others/en/instruments/product-finder/type/quantum_computing_systems>`_:
```

Comment:
I was generally using Qibolab instead of ``Qibolab`` across the documentation, including the QM section right below this, but I am not sure if we have decided on a convention.